memphis/templates/memphis-creds.yaml

```diff
@@ -1,17 +1,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: memphis-creds
+  name: {{ .Values.memphis.creds.secretConfig.name }}
```

Comment:
@madestreel In case the secret already exist in the namespace and its not helm generated the installation fails.

Reply:
Could you provide the error? When I tested it worked fine... But it is true that I didn't test with other keys than the defaults one inside the secret

Reply:
I've tried to use pre-created secret and this is the output:
`Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists. Unable to continue with install: Secret "external-creds" in namespace "memphis" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "my-memphis"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "memphis"`

Reply:
I may be mistaken but it seems that the error is more about the fact that the secret cannot be imported to the release more than the chart trying to create a secret that already exist.

Reply:
It can't be imported and we don't want so, we want to use an existing pre-configured secret with all the values. 
Can you explain a little bit more about the use case you checked?

Reply:
Sorry I was not working yesterday so I had not really the chance to test. I will do that today